tools/ci_build/github/azure-pipelines/templates/win-web-ci.yml

```diff
@@ -123,12 +126,13 @@ jobs:
      node -e "a=require('child_process').execSync('git diff --name-only').toString();if(a)throw new Error('Following documents are not up-to-date: (did you run \"npm run build:doc\"?)\n'+a)"
     workingDirectory: '$(Build.SourcesDirectory)\js\web'
     displayName: 'Check out of dated documents'
-  - script: |
-     if exist "C:\local\data\onnx\test\data\node\__generated_onnx_node_tests" (
-      xcopy C:\local\data\onnx\test\ $(Build.SourcesDirectory)\js\test\ /E
-     )
-    workingDirectory: '$(Build.SourcesDirectory)\js\web'
-    displayName: 'Copy pre-downloaded test data'
+  - task: Cache@2
+    inputs:
+      # once the test data is changed, we need to update the cache key.
+      key: onnxtestdata | opset18
```

Comment:
can we use something generated automatically as key , instead of having to update every time.

for example, can we use the commit SHA of git submodule onnx as key?